src/thermo/src/4C_thermo_timint.hpp

```diff
@@ -531,8 +536,36 @@ namespace Thermo
     //! @name Printing and output
     //@{
 
+    struct OptionsThermoRuntimeOutput
+    {
+      /// whether to write pressure output
+      bool output_temperature_state = false;
+
+      /// whether to write heatflux output
+      bool output_heatflux_state = false;
+
+      /// whether to write temperature gradient output
+      bool output_tempgrad_state = false;
+
+      /// whether to write energy output
+      bool output_energy_state = false;
+
+      /// whether to write the owner of elements
+      bool output_element_owner = false;
+
+      /// whether to write the element GIDs
+      bool output_element_gid = false;
+
+      /// whether to write the node GIDs
+      bool output_node_gid = false;
+    };
+
     std::shared_ptr<Core::IO::DiscretizationWriter> output_;  //!< binary output
-    bool printlogo_;                                          //!< true: enjoy your cuppa
+    std::shared_ptr<Core::IO::DiscretizationVisualizationWriterMesh>
```

Comment:
A `std::optional` would probably convey more clearly that this member is not always needed.

Reply:
I agree, makes it more straightforward to understand.